search-enterprise/latest/using/unsupported.md

```diff
@@ -57,8 +55,8 @@ The following features, which were supported with Search and Insight Engine 2.x
 
 ### Path Indexing
 
+* Secondary parents
 * Secondary paths (paths including secondary parents)
```

Comment:
That is not true anymore. With ESC 4.0.0 we are introducing secondary paths and parents.

Reply:
Removed note